# Awesome List Updates on Sep 24, 2019

19 awesome lists updated today.

[üè† Home](/README.md) ¬∑ [üîç Search](https://www.trackawesomelist.com/search/) ¬∑ [üî• Feed](https://www.trackawesomelist.com/rss.xml) ¬∑ [üìÆ Subscribe](https://trackawesomelist.us17.list-manage.com/subscribe?u=d2f0117aa829c83a63ec63c2f&id=36a103854c) ¬∑ [‚ù§Ô∏è  Sponsor](https://github.com/sponsors/theowenyoung)



## [1. Awesome Actions](/content/sdras/awesome-actions/README.md)

### Static Analysis / Linting

*   [Run TSLint with status checks and file diff annotations (‚≠ê34)](https://github.com/mooyoul/tslint-actions)

## [2. Awesome Cybersecurity Blueteam](/content/fabacab/awesome-cybersecurity-blueteam/README.md)

### Phishing awareness and reporting / Firewall appliances or distributions

*   [Phishing Intelligence Engine (PIE) (‚≠ê179)](https://github.com/LogRhythm-Labs/PIE) - Framework that will assist with the detection and response to phishing attacks.

## [3. Awesome Parasite](/content/ecohealthalliance/awesome-parasite/README.md)

### Citizen Science Projects

*   [Garden Wildlife Health](https://www.gardenwildlifehealth.org) - This is a UK collaborative project between the Zoological Society of London (ZSL), the British Trust for Ornithology (BTO), Froglife and the Royal Society for the Protection of Birds (RSPB) that historically has been running since the 1980's (previously as Garden Bird Health initiative and Frog Mortality Project). Citizens can submit sickness and mortality reports, as well as animal wildlife samples, for wild birds, amphibians, reptiles, and hedgehogs.
*   [Project Monarch Health](https://www.monarchparasites.org/) - A citizen science project running since 2006 to test adult monarch butterflies from the U.S. and Canada for the protozoan parasite *Ophryocystis elektroscirrha*. Participants submit spore samples collected from adult abdomems with tape for laboratory assessment at University of Georgia.

## [4. Awesome Ros2](/content/fkromer/awesome-ros2/README.md)

### Companies / Robotics Capture the Flag (RCTF)

*   [Apex.AI](https://www.apex.ai/) - "Safe and certified software for autonomous mobility".
*   [AutonomouStuff](https://autonomoustuff.com) - "The world leader in autonomy systems and solutions".
*   [MathWorks](https://de.mathworks.com/help/ros/index.html) - ROS Toolbox.

## [5. Awesome Static Website Services](/content/agarrharr/awesome-static-website-services/README.md)

### Forms / Normal Forms

*   [StaticKit](https://statickit.com) - Modern forms for static sites, with native support for React.

## [6. Awesome Bioie](/content/caufieldjh/awesome-bioie/README.md)

### Research Overviews / Pre-LLM Overviews

*   [Biomedical Informatics on the Cloud: A Treasure Hunt for Advancing Cardiovascular Medicine](https://www.ahajournals.org/doi/full/10.1161/CIRCRESAHA.117.310967) - An overview of how BioIE and bioinformatics workflows can be applied to questions in cardiovascular health and medicine research.
*   [Clinical information extraction applications: A literature review](https://www.sciencedirect.com/science/article/pii/S1532046417302563) - A review of clinical IE papers published as of September 2016. From Mayo Clinic group (see below).
*   [Mining Electronic Health Records (EHRs): A Survey](https://arxiv.org/abs/1702.03222) - A review of the methods and philosophy behind mining electronic health records, including using them for adverse event detection. See Table 2 for a list of relevant papers as of mid-2017.

### Organizations / Pre-LLM Overviews

*   [AMIA](https://www.amia.org/) - Many‚Äîbut certainly not all‚Äîindividuals studying biomedical informatics are members of the American Medical Informatics Association. AMIA publishes a journal, JAMIA (see below).
*   [IMIA](https://imia-medinfo.org/) - The International Medical Informatics Association. Publishes the IMIA Yearbook of Medical Informatics.

### Journals and Events / Journals

*   [Database](https://academic.oup.com/database) - Its subtitle is "The Journal of Biological Databases and Curation". Open access.
*   [JAMIA](https://academic.oup.com/jamia) - The Journal of the American Medical Informatics Association. Concerns "articles in the areas of clinical care, clinical research, translational science, implementation science, imaging, education, consumer health, public health, and policy".
*   [JBI](https://www.sciencedirect.com/journal/journal-of-biomedical-informatics) - The Journal of Biomedical Informatics. Not open access by default, though it does have an open-access "X" version.
*   [Scientific Data](https://www.nature.com/sdata/) - An open-access Springer Nature journal publishing "descriptions of scientifically valuable datasets, and research that advances the sharing and reuse of scientific data".

### Journals and Events / Conferences and Other Events

*   [ACM-BCB](http://acm-bcb.org/) - The ACM Conference on Bioinformatics, Computational Biology, and Health Informatics. Held annually since 2010.
*   [BIBM](http://ieeebibm.org/BIBM2019/) - The IEEE International Conference on Bioinformatics and Biomedicine.
*   [ISMB](https://www.iscb.org/about-ismb) - The International Conference on Intelligent Systems for Molecular Biology is an annual conference hosted by the International Society for Computational Biology since 1993. Much of its focus has concerned bioinformatics and computational biology without an explicit clinical focus, though it has included an increasing amount of text mining content (e.g., the 2019 meeting included a [full-day special session on Text Mining for Biology and Healthcare](http://cosi.iscb.org/wiki/TextMining:Home)). The meeting is combined with that of the European Conference on Computational Biology (ECCB) on odd-numbered years.
*   [PSB](https://psb.stanford.edu/) - The Pacific Symposium on Biocomputing.

### Tutorials / Pre-LLM Guides, Lectures, and Courses

*   [Getting Started in Text Mining](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0040020) - A brief introduction to bio-text mining from Cohen and Hunter. More than ten years old but still quite relevant. See also an [earlier paper by the same authors](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1702322/).
*   [Biomedical Literature Mining](https://link.springer.com/book/10.1007/978-1-4939-0709-0) - A (non-free) volume of Methods in Molecular Biology from 2014. Chapters covers introductory principles in text mining, applications in the biological sciences, and potential for use in clinical or medical safety scenarios.
*   [VIB text mining and curation training](https://www.bits.vib.be/training-list/111-bits/training/previous-trainings/183-text-mining) - This training workshop happenened in 2013 but the slides are still online.

### Code Libraries / Pre-LLM Guides, Lectures, and Courses

*   [Bio-SCoRes (‚≠ê9)](https://github.com/kilicogluh/Bio-SCoRes) - [paper](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0148538) - A framework for biomedical coreference resolution.
*   [medaCy (‚≠ê424)](https://github.com/NLPatVCU/medaCy) - A system for building predictive medical natural language processing models. Built on the [spaCy](https://spacy.io/) framework.

### Tools, Platforms, and Services / Repos for Specific Datasets

*   [cTAKES](https://ctakes.apache.org/) - [paper](https://academic.oup.com/jamia/article/17/5/507/830823) - [code (‚≠ê39)](https://github.com/apache/ctakes) - A system for processing the text in electronic medical records. Widely used and open source.
*   [CLAMP](https://clamp.uth.edu/) - [paper](https://academic.oup.com/jamia/article/25/3/331/4657212) - A natural language processing toolkit intended for use with the text in clinical reports. Check out their [live demo](https://clamp.uth.edu/clampdemo.php) first to see what it does. Usable at no cost for academic research.
*   [DeepPhe (‚≠ê29)](https://github.com/DeepPhe/DeepPhe-Release) - A system for processing documents describing cancer presentations. Based on cTAKES (see above).
*   [DNorm](https://www.ncbi.nlm.nih.gov/research/bionlp/Tools/dnorm/) - [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3810844/) - A method for disease normalization, i.e., linking mentions of disease names and acronyms to unique concept identifiers. Downloadable version includes the NCBI Disease Corpus and BC5CDR (see Annotated Text Data below).
*   [PubTator Central](https://www.ncbi.nlm.nih.gov/research/pubtator/) - [paper](https://academic.oup.com/nar/article/47/W1/W587/5494727) - A web platform that identifies five different types of biomedical concepts in PubMed articles and PubMed Central full texts. The full annotation sets are downloadable (see [Annotated Text Data](#annotated-text-data) below).
*   [Pubrunner (‚≠ê41)](https://github.com/jakelever/pubrunner) - A framework for running text mining tools on the newest set(s) of documents from PubMed.
*   [TaggerOne](https://www.ncbi.nlm.nih.gov/research/bionlp/Tools/taggerone/) - [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5018376/) - Performs concept normalization (see also DNorm above). Can be trained for specific concept types and can perform NER independent of other normalization functions.

### Tools, Platforms, and Services / Annotation Tools

*   [Anafora (‚≠ê238)](https://github.com/weitechen/anafora) - [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5657237/) - An annotation tool with adjudication and progress tracking features.

### Techniques and Models / BERT models

*   [BioBERT (‚≠ê643)](https://github.com/naver/biobert-pretrained) - [paper](https://arxiv.org/abs/1901.08746) - [code (‚≠ê1.9k)](https://github.com/dmis-lab/biobert) - A PubMed and PubMed Central-trained version of the [BERT language model](https://arxiv.org/abs/1810.04805).

### Techniques and Models / Other models

*   [Flair embeddings from PubMed (‚≠ê14k)](https://github.com/zalandoresearch/flair/pull/519) - A language model available through the Flair framework and embedding method. Trained over a 5% sample of PubMed abstracts until 2015, or > 1.2 million abstracts in total.

### Techniques and Models / Text Embeddings

*   [BioASQword2vec](http://bioasq.org/news/bioasq-releases-continuous-space-word-vectors-obtained-applying-word2vec-pubmed-abstracts) - [paper](http://bioasq.lip6.fr/info/BioASQword2vec/) - Qord embeddings derived from biomedical text (>10 million PubMed abstracts) using the popular [word2vec](https://code.google.com/archive/p/word2vec/) tool.
*   [BioWordVec](https://figshare.com/articles/Improving_Biomedical_Word_Embeddings_with_Subword_Information_and_MeSH_Ontology/6882647) - [paper](https://www.nature.com/articles/s41597-019-0055-0) - [code (‚≠ê141)](https://github.com/ncbi-nlp/BioWordVec) - Word embeddings derived from biomedical text (>27 million PubMed titles and abstracts), including subword embedding model based on MeSH.

### Datasets / Biomedical Text Sources

*   [PubMed Central Open Access Subset](https://www.ncbi.nlm.nih.gov/pmc/tools/openftlist/) - A set of PubMed Central articles usable under licenses other than traditional copyright, though the exact licenses vary by publication and source. Articles are available as PDF and XML.

### Datasets / Annotated Text Data

*   [SPL-ADR-200db](https://bionlp.nlm.nih.gov/tac2017adversereactions/) - [paper](https://www.nature.com/articles/sdata20181) - A pilot dataset containing standardised information, and annotations of occurence in text, about \~5,000 known adverse reactions for 200 FDA-approved drugs.
*   [BioCreAtIvE 1](https://sourceforge.net/projects/biocreative/files/) - [paper](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-6-S1-S1) - 15,000 sentences (10,000 training and 5,000 test) annotated for protein and gene names. 1,000 full text biomedical research articles annotated with protein names and Gene Ontology terms.
*   [BioCreAtIvE 2](https://sourceforge.net/projects/biocreative/files/) - [paper](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2008-9-s2-s1) - 15,000 sentences (10,000 training and 5,000 test, different from the first corpus) annotated for protein and gene names. 542 abstracts linked to EntrezGene identifiers. A variety of research articles annotated for features of protein‚Äìprotein interactions.
*   [BioCreAtIvE V CDR Task Corpus (BC5CDR)](https://biocreative.bioinformatics.udel.edu/accounts/login/?next=/resources/corpora/biocreative-v-cdr-corpus/) - [paper](https://academic.oup.com/database/article/doi/10.1093/database/baw068/2630414) - 1,500 articles (title and abstract) published in 2014 or later, annotated for 4,409 chemicals, 5,818 diseases and 3116 chemical‚Äìdisease interactions. Requires registration.
*   [BioCreative VI CHEMPROT Corpus](https://biocreative.bioinformatics.udel.edu/resources/corpora/chemprot-corpus-biocreative-vi/#chemprot-corpus-biocreative-vi:downloads) - [paper](https://pdfs.semanticscholar.org/eed7/81f498b563df5a9e8a241c67d63dd1d92ad5.pdf) - >2,400 articles annotated with chemical-protein interactions of a variety of relation types. Requires registration.
*   [n2c2 (formerly i2b2) Data](https://portal.dbmi.hms.harvard.edu/projects/n2c2-nlp/) - The Department of Biomedical Informatics (DBMI) at Harvard Medical School manages data for the National NLP Clinical Challenges and the Informatics for Integrating Biology and the Bedside challenges running since 2006. They require registration before access and use. Datasets include a variety of topics. See the [list of data challenges](https://portal.dbmi.hms.harvard.edu/data-challenges/) for individual descriptions.
*   [NCBI Disease Corpus](https://www.ncbi.nlm.nih.gov/CBBresearch/Dogan/DISEASE/) - [paper](https://www.sciencedirect.com/science/article/pii/S1532046413001974) - A corpus of 793 biomedical abstracts annotated with names of diseases and related concepts from MeSH and [OMIM](https://omim.org/).
*   [PubTator Central datasets](https://www.ncbi.nlm.nih.gov/research/pubtator/) - [paper](https://academic.oup.com/nar/article/47/W1/W587/5494727) - Accessible through a RESTful API or FTP download. Includes annotations for >29 million abstracts and ‚àº3 million full text documents.

### Datasets / Protein-protein Interaction Annotated Corpora

*   [HPRD50](http://corpora.informatik.hu-berlin.de/corpora/brat2bioc/hprd50_bioc.xml.zip) - [paper](https://academic.oup.com/bioinformatics/article/23/3/365/236564) - 50 scientific abstracts referenced by the Human Protein Reference Database, annotated for PPI.

### Datasets / Other Datasets

*   [MIMIC-III](https://mimic.physionet.org/) - [paper](https://www.nature.com/articles/sdata201635) - Deidentified health data from \~60,000 intensive care unit admissions. Requires completion of an online training course (CITI training) and acceptance of a data use agreement prior to use.
*   [MIMIC-CXR](https://physionet.org/content/mimic-cxr/2.0.0/) - The MIMIC Chest X-Ray database. Contains more than 377,000 radiographic images and accompanying free-text radiology reports. As with MIMIC-III, requires acceptance of a data use agreement.
*   [UMLS Knowledge Sources](https://www.nlm.nih.gov/research/umls/licensedcontent/umlsknowledgesources.html) - [reference manual](https://www.ncbi.nlm.nih.gov/books/NBK9676/) - A large and comprehensive collection of biomedical terminology and identifiers, as well as accompanying tools and scripts. Depending on your purposes, the single file MRCONSO.RRF may be sufficient, as this file contains unique identifiers and names for all concepts in the UMLS Metathesaurus. See also the Ontologies and Controlled Vocabularies section below.

### Ontologies and Controlled Vocabularies / Other Datasets

*   [Disease Ontology](http://www.disease-ontology.org/) - [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4383880/) - An ontology of human diseases. Has cross-links to MeSH, ICD, NCI Thesaurus, SNOMED, and OMIM. Public domain. Available on [GitHub (‚≠ê329)](https://github.com/DiseaseOntology/HumanDiseaseOntology) and on the [OBO Foundry](http://www.obofoundry.org/ontology/doid.html).
*   [SPECIALIST Lexicon](https://lexsrv3.nlm.nih.gov/Specialist/Summary/lexicon.html) - [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2247735/) - A general English lexicon that includes many biomedical terms. Updated yearly since 1994 and still updated as of 2019. Part of UMLS but does not require UTS account to download.

### Data Models / Other Datasets

*   [Biolink](https://biolink.github.io/biolink-model/) - [code (‚≠ê169)](https://github.com/biolink/biolink-model) - A data model of biological entities. Provided as a [YAML](https://yaml.org/) file.
*   [BioUML](http://wiki.biouml.org/index.php/BioUML) - [paper](https://academic.oup.com/nar/article/47/W1/W225/5498754) - An architecture for biomedical data analysis, integration, and visualization. Conceptually based on the visual modeling language [UML](https://www.uml.org/what-is-uml.htm).

## [7. Awesome Mobile Web Development](/content/myshov/awesome-mobile-web-development/README.md)

### Articles and Documentation

*   [How Web Content Can Affect Power Usage](https://webkit.org/blog/8970/how-web-content-can-affect-power-usage/) - What you can do with web app to achieve better battery life.
*   [The joy and challenge of developing for KaiOS](https://nolanlawson.com/2019/09/22/the-joy-and-challenge-of-developing-for-kaios/) - How to start developing for KaiOS.

## [8. Awesome Kde](/content/francoism90/awesome-kde/README.md)

### Internet & Networking

*   [Firefox KDE](https://build.opensuse.org/package/show/mozilla:Factory/MozillaFirefox) - Standalone web browser from mozilla.org with OpenSUSE patch, integrate better with KDE.

### Plasma desktop / Themes

*   [Breath](https://gitlab.manjaro.org/artwork/themes/breath) - Breath Theme for Manjaro Linux (Plasma 5).

## [9. Awesome Piracy](/content/Igglybuff/awesome-piracy/README.md)

### Stremio / Third Party Hosts

*   [A complete curated list of all working Stremio Add-ons](https://www.reddit.com/r/StremioAddons/comments/ahd6gk/sticky_stremio_addons_list/)

## [10. Awesome Ipfs](/content/ipfs/awesome-ipfs/README.md)

### Tools

*   [go-orbit-db (‚≠ê427)](https://github.com/berty/go-orbit-db) - This is a Golang port of OrbitDB that intends to be fully compatible with the original JavaScript version. OrbitDB is a serverless, distributed, peer-to-peer database.

## [11. Awesome Newsletters](/content/zudochkin/awesome-newsletters/README.md)

### Swift / Svelte

*   [The Swift by Sundell](https://www.swiftbysundell.com/newsletter/). Introducing the Swift by Sundell monthly newsletter!

### PHP / Svelte

*   [Freek.dev](https://freek.dev/newsletter/). A bi weekly newsletter on modern PHP and Laravel.

### Devops / Svelte

*   [StatusCode Weekly](https://weekly.statuscode.com/). A weekly newsletter on Web operations, infrastructure, performance, and tooling, from the browser down to the metal.

### Cloud Development / Svelte

*   [Last Week in AWS](https://www.lastweekinaws.com/). A weekly roundup of news from Amazon's cloud ecosystem.

### Business/Finance / Svelte

*   [FBA monthly](https://www.fbamonthly.com/). FBA Monthly is an across-the-board summary of the month's most important news articles and blog posts regarding Amazon businesses. Only actual news, solutions, strategies, seller-to-seller tips and much more.

### Entrepreneurship / Svelte

*   [Founders Grid](https://smash.vc/founders-grid/). A weekly newsletter for startup founders that rounds up the best resources from startup ecosystem.

## [12. Awesome Roslyn](/content/ironcev/awesome-roslyn/README.md)

### Open Source Projects

*   [FlubuCore (‚≠ê816)](https://github.com/dotnetcore/FlubuCore) - Cross platform build automation tool for building projects and executing deployment scripts using C# code.

## [13. Awesome Ocaml](/content/ocaml-community/awesome-ocaml/README.md)

### Internationalization

*   [Camomile (‚≠ê125)](https://github.com/yoriyuki/Camomile/) ‚Äî A Unicode library for OCaml.
*   [ocaml-m17n (‚≠ê50)](https://github.com/whitequark/ocaml-m17n) ‚Äî Multilingualization for OCaml source code. Allows using Unicode identifiers in OCaml source code.
*   [Uucd (‚≠ê17)](https://github.com/dbuenzli/uucd) ‚Äî Unicode character database decoder for OCaml.
*   [Uucp (‚≠ê21)](https://github.com/dbuenzli/uucp) ‚Äî Unicode character properties for OCaml.
*   [Uunf (‚≠ê21)](https://github.com/dbuenzli/uunf) ‚Äî Unicode text normalization for OCaml.
*   [Uuseg (‚≠ê23)](https://github.com/dbuenzli/uuseg) ‚Äî Unicode text segmentation for OCaml.
*   [Uutf (‚≠ê32)](https://github.com/dbuenzli/uutf) ‚Äî Non-blocking streaming Unicode codec for OCaml.

## [14. Awesome React Native](/content/jondot/awesome-react-native/README.md)

### Seeds / Navigation Demos

*   [:octopus: React Native Boilerplate ‚òÖ551 (‚≠ê3.3k)](https://github.com/thecodingmachine/react-native-boilerplate) - React Native boilerplate that promotes a solid architecture via separation of concerns.

## [15. Awesome Flutter](/content/Solido/awesome-flutter/README.md)

### Auth / Image Picker

*   [Flutter AppAuth (‚≠ê279)](https://github.com/MaikuB/flutter_appauth) <!--stargazers:MaikuB/flutter_appauth--> - Plugin that provides a wrapper around the AppAuth iOS and Android SDKs by [Michael Bui](https://github.com/MaikuB)

### üá¶üá∫ Australia / Misc

*   Slack [GDG Sydney](https://gdg-sydney.slack.com)
*   Meetup [GDG Sydney](https://www.meetup.com/gdgsydney)
*   Twitter [Flutter Sydney](https://twitter.com/FlutterSydney)

## [16. Awesome Clojurescript](/content/hantuzun/awesome-clojurescript/README.md)

### Resources / Books

*   [ClojureScript Unraveled](https://leanpub.com/clojurescript-unraveled) ‚Äì An open source book about the ClojureScript language that covers all the language features, how to use the compiler and the tooling for building applications and libraries.
*   [ClojureScript: Up and Running](https://shop.oreilly.com/product/0636920025139.do) ‚Äì An introduction to ClojureScript written by big guns of Clojure Stuart Sierra and Luke VanderHart.
*   [Etudes for ClojureScript](https://shop.oreilly.com/product/0636920043584.do) ‚Äì A hands-on-book of 30 companion exercises or √©tudes for introducing ClojureScript.
*   [Learning ClojureScript](https://www.packtpub.com/web-development/learning-clojurescript) ‚Äì Master the art of agile single page web application development with ClojureScript.
*   [Transforming Data with ClojureScript](https://langintro.com/cljsbook) ‚Äì A beginner's guide to ClojureScript with interactive examples and exercises, "focusing on the main task that programming does‚Äîtransforming data".

### Resources / Courses

*   [ClojureScript Koans](http://clojurescriptkoans.com/) ‚Äì A great way to start learning Clojure and ClojureScript on the web with interactive challanges.
*   [Lambda Island](https://lambdaisland.com/) ‚Äì A full-stack web development course including front-end with ClojureScript, back-end with Clojure, language essentials, security, internet standards and system administration.
*   [Learn Reagent](https://www.learnreagent.com/) ‚Äì An introduction to ClojureScript with Reagent for developers. 1 hour free content is available. [learn re-frame](https://www.learnreframe.com/) is its follow up.
*   [Learning ClojureScript](https://purelyfunctional.tv/) ‚Äì A 2h 11m long introduction to Clojurescript by O'Really Online Learning. Also available on [Udemy](https://www.udemy.com/course/learning-clojurescript/).

### Resources / Videos

*   [ClojureScript for Skeptics](https://www.youtube.com/watch?v=gsffg5xxFQI) ‚Äì Derek Slager's talk for Clojure Conj 2015 where he is Discussing the many reasons why ClojureScript is in fact a very pragmatic language to consider for web development.
*   [Interactive programming Flappy Bird in ClojureScript](https://www.youtube.com/watch?v=KZjFVdU8VLI) ‚Äì A 5 minutes video from 2014 demostrating the power of ClojureScript's developer experience.

### Awesome ClojureScript / Code Analysis

*   [kibit (‚≠ê1.7k)](https://github.com/jonase/kibit) ‚Äì Static code analyzer to find patterns of code that could be rewritten with a more idiomatic function or macro.

### Awesome ClojureScript / Development

*   [Ambly (‚≠ê535)](https://github.com/omcljs/ambly) ‚Äì A ClojureScript REPL into iOS JavaScriptCore.
*   [Instaparse (‚≠ê89)](https://github.com/lbradstreet/instaparse-cljs) ‚Äì It aims to be the simplest way to build parsers in ClojureScript.
*   [lein-cljsbuild (‚≠ê1.1k)](https://github.com/emezeske/lein-cljsbuild) ‚Äì A Leiningen plugin to make ClojureScript development easy.
*   [shadow-cljs (‚≠ê2k)](https://github.com/thheller/shadow-cljs) ‚Äì ClojureScript compilation made easy
*   [Truss (‚≠ê274)](https://github.com/ptaoussanis/truss) ‚Äì An opinionated assertions API for Clojure/ClojureScript.

### Awesome ClojureScript / Document Object Model

*   [Respo (‚≠ê232)](https://github.com/mvc-works/respo) ‚Äì A responsive DOM library, inspired by React.

### Awesome ClojureScript / Internationalization

*   [Tempura (‚≠ê242)](https://github.com/ptaoussanis/tempura) ‚Äì A Clojure(Script) i18n library, succeeding Tower

### Awesome ClojureScript / [React.js](https://facebook.github.io/react/)   Interface

*   [cljsx (‚≠ê10)](https://github.com/peterhudec/cljsx) ‚Äì [JSX](https://reactjs.org/docs/introducing-jsx.html) for Clojure and ClojureScript, also works with [Inferno](https://infernojs.org), [Nerv](https://nerv.aotu.io/), [Preact](https://preactjs.com/), [Snabbdome (‚≠ê11k)](https://github.com/snabbdom/snabbdom) and others.
*   [Om (‚≠ê6.7k)](https://github.com/omcljs/om) ‚Äì A powerful interface to React, makes use of its object oriented structures.
*   [Quiescent (‚≠ê609)](https://github.com/levand/quiescent) ‚Äì It favors functional style, fully stateless.
*   [Reagent](http://reagent-project.github.io/) ‚Äì Minimalistic, feature complete.
*   [Rum (‚≠ê1.7k)](https://github.com/tonsky/rum) ‚Äì Decomplected, extensible, simple.

### Awesome ClojureScript / Reactive Programming

*   [Manifold-cljs (‚≠ê46)](https://github.com/dm3/manifold-cljs) ‚Äì A port of [Manifold (‚≠ê948)](https://github.com/ztellman/manifold) to ClojureScript.

### Awesome ClojureScript / Web Framework & Template

*   [Fulcro](http://fulcrologic.github.io/fulcro) ‚Äì A library for development of single-page full-stack web applications in clj/cljs.
*   [Mr-Clean](https://bitbucket.org/sonwh98/mr-clean) ‚Äì A reagent compatible library without react.js dependency.
*   [Precept (‚≠ê642)](https://github.com/CoNarrative/precept) ‚Äì A declarative programming framework.

## [17. Android Security Awesome](/content/ashishb/android-security-awesome/README.md)

### Tools / Reverse Engineering

*   [Simplify Android deobfuscator (‚≠ê4.5k)](https://github.com/CalebFenton/simplify)
*   [Andromeda (‚≠ê704)](https://github.com/secrary/Andromeda) - Another basic command-line reverse engineering tool

## [18. Awesome Python Data Science](/content/krzjoa/awesome-python-data-science/README.md)

### Deep Learning / TensorFlow

*   [tensorpack (‚≠ê6.3k)](https://github.com/ppwwyyxx/tensorpack) - A Neural Net Training Interface on TensorFlow. <img height="20" src="https://github.com/krzjoa/awesome-python-data-science/raw/master/img/tf_big2.png" alt="sklearn">

## [19. Awesome Actions on Google](/content/ravirupareliya/awesome-actions-on-google/README.md)

### Open source code

*   [karaoke (‚≠ê11)](https://github.com/amdcaruso/karaoke) - [Amanda Cavallaro](https://github.com/amdcaruso) - Conversational App to buid your own karaoke using GCP, YouTube API and Actions on Google.

---

- Prev: [Sep 25, 2019](/content/2019/09/25/README.md)
- Next: [Sep 23, 2019](/content/2019/09/23/README.md)