# Awesome Computational Biology Overview

Awesome list of computational biology.

[üè† Home](/README.md) ¬∑ [üî• Feed](https://www.trackawesomelist.com/inoue0426/awesome-computational-biology/rss.xml) ¬∑ [üìÆ Subscribe](https://trackawesomelist.us17.list-manage.com/subscribe?u=d2f0117aa829c83a63ec63c2f&id=36a103854c) ¬∑ [‚ù§Ô∏è  Sponsor](https://github.com/sponsors/theowenyoung) ¬∑ [üò∫ inoue0426/awesome-computational-biology](https://github.com/inoue0426/awesome-computational-biology) ¬∑ ‚≠ê 113 ¬∑ üè∑Ô∏è Miscellaneous

[ [Daily](/content/inoue0426/awesome-computational-biology/README.md) / [Weekly](/content/inoue0426/awesome-computational-biology/week/README.md) / Overview ]

---

# Awesome Computational Biology [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)

A curated collection of databases, software, and papers related to computational biology.

> Computational biology involves the development and application of data-analytical and theoretical methods, mathematical modelling and computational simulation techniques to the study of biological, ecological, behavioural, and social systems. ‚Äî [Wikipedia](https://en.wikipedia.org/wiki/Computational_biology)

***

## Table of Contents

*   [Databases](#databases)
    *   [scRNA](#scrna)
    *   [Compound](#compound)
    *   [Pathway](#pathway)
    *   [Mass Spectra](#mass-spectra)
    *   [Protein](#protein)
    *   [Genome](#genome)
    *   [Disease](#disease)
    *   [Interaction](#interaction)
        *   [Drug-Gene Interaction](#drug-gene-interaction)
        *   [Drug (Cell Line) Response](#drug-cell-line-response)
        *   [Chemical-Protein Interaction](#chemical-protein-interaction)
        *   [Protein-Protein Interaction](#protein-protein-interaction)
        *   [Knowledge Graph](#knowledge-graph)
    *   [Clinical Trial](#clinical-trial)
*   [API](#api)
*   [Preprocessing Tools](#preprocessing-tools)
*   [Machine Learning Tasks and Models](#machine-learning-tasks-and-models)
    *   [Drug Response Prediction](#drug-response-prediction)
    *   [Drug Repurposing](#drug-repurposing)
    *   [Drug Target Interaction](#drug-target-interaction)
    *   [Compound-Protein Interaction](#compound-protein-interaction)
    *   [Pre-trained Embedding](#pre-trained-embedding)
    *   [LLM for Biology](#llm-for-biology)
    *   [Foundation Models](#foundation-models)

***

## Databases

### scRNA

*   [Gene Expression Omnibus](https://www.ncbi.nlm.nih.gov/geo/) ‚Äî Public functional genomics database.
*   [Single Cell PORTAL](https://singlecell.broadinstitute.org/single_cell) ‚Äî Public database for single-cell RNA.
*   [Single Cell Expression Atlas](https://www.ebi.ac.uk/gxa/sc/home) ‚Äî Public database for single-cell RNA.

### Compound

*   [PubChem](https://pubchem.ncbi.nlm.nih.gov/) ‚Äî One of the largest chemical databases (compounds, genes, and proteins).
*   [ChEBI](https://www.ebi.ac.uk/chebi/) ‚Äî Database focused on small chemical compounds.
*   [ChEMBL](https://www.ebi.ac.uk/chembl/) ‚Äî Bioactive molecules with drug-like properties.
*   [ChemSpider](http://www.chemspider.com/) ‚Äî Chemical structure database.
*   [KEGG COMPOUND](https://www.genome.jp/kegg/compound/) ‚Äî Collection of small molecules and biopolymers.
*   [LIPID MAPS](https://www.lipidmaps.org/databases/lmsd/overview) ‚Äî Database of lipids.
*   [Rhea](https://www.rhea-db.org/) ‚Äî Database of chemical reactions.
*   [Drug Repurposing Hub](https://repo-hub.broadinstitute.org/repurposing#download-data) ‚Äî Collections of drug repurposing data (drug, MoA, target, etc).
*   [Therapeutic Target Database](https://idrblab.net/ttd/full-data-download) ‚Äî Drug-target, target-disease, and drug-disease datasets.
*   [ZINC ligand discovery database](https://zinc.docking.org/) ‚Äî Free database of commercially-available compounds for virtual screening.
*   [MoleculeNet](http://moleculenet.ai/) ‚Äî Benchmark datasets for molecular machine learning.

### Pathway

*   [PathwayCommons](https://www.pathwaycommons.org/) ‚Äî Database of pathways and interactions.
*   [KEGG PATHWAY](https://www.genome.jp/kegg/pathway.html) ‚Äî Collection of pathway maps.
*   [WikiPathways](https://wikipathways.org/) ‚Äî Database of biological pathways.
*   [Reactome](https://reactome.org/) ‚Äî Expert-curated, peer-reviewed pathway database with detailed reaction mechanisms.
*   [BioCyc](https://biocyc.org/) ‚Äî Collection of pathway/genome databases across thousands of organisms.
*   [SIGNOR](https://signor.uniroma2.it/) ‚Äî Database of causal signaling interactions and pathways.
*   [MSigDB (Molecular Signatures Database)](https://www.gsea-msigdb.org/gsea/msigdb) ‚Äî Curated gene sets derived from pathways and biological processes.

### Mass Spectra

*   [MassBank](http://www.massbank.jp/) ‚Äî Open source databases and tools for mass spectrometry reference spectra.
*   [MoNA MassBank of North America](https://mona.fiehnlab.ucdavis.edu/) ‚Äî Meta-database of metabolite mass spectra, metadata, and associated compounds.

### Protein

*   [THE HUMAN PROTEIN ATLAS](https://www.proteinatlas.org/) ‚Äî Comprehensive human protein database (cells, tissues, organs).
*   [PROTEIN DATA BANK (PDB)](https://www.rcsb.org/) ‚Äî 3D structures of proteins, nucleic acids, complexes.
*   [UniProt](https://www.uniprot.org/) ‚Äî Functional information on proteins.
*   [AlphaFold Protein Structure Database](https://alphafold.ebi.ac.uk/api-docs) ‚Äî 3D protein structure predictions.
*   [RCSB Protein Data Bank](https://www.rcsb.org/) ‚Äî Repository for structural data of biological molecules.
*   [Critical Assessment of Structure Prediction (CASP)](https://predictioncenter.org/) ‚Äî Assessing methods for protein structure prediction.
*   [Uniclust](https://uniclust.mmseqs.com/) ‚Äî Clustered protein sequence databases.
*   [CATH database](https://www.cathdb.info/) ‚Äî Hierarchical classification of protein domain structures.

### Genome

*   [Human Genome Resources at NCBI](https://www.ncbi.nlm.nih.gov/projects/genome/guide/human/index.shtml) ‚Äî Database for genomics, proteomics, transcriptomics, and systems biology.
*   [GenBank](https://www.ncbi.nlm.nih.gov/genbank/) ‚Äî NCBI's database of genetic sequences.
*   [UCSC Genome Browser](https://genome.ucsc.edu/) ‚Äî UCSC's genome browser.
*   [cBioPortal](https://www.cbioportal.org/) ‚Äî Cancer genomics database; aggregating many patient datasets.
*   [10x Genomics Dataset](https://www.10xgenomics.com/resources/datasets) ‚Äî Collection of single-cell datasets.
*   [The Genotype-Tissue Expression (GTEx)](https://gtexportal.org/home/) ‚Äî Human gene expression and regulation resource.
*   [Dependency Map (DepMap)](https://depmap.org/portal/) ‚Äî CRISPR-Cas9 screens in cancer cell lines.
*   [Catalogue Of Somatic Mutations In Cancer (COSMIC)](https://cancer.sanger.ac.uk/cosmic) ‚Äî Resource on somatic mutations in cancers.
*   [MGnify](https://www.ebi.ac.uk/metagenomics/) ‚Äî Resource for metagenomic and metatranscriptomic data.
*   [JASPAR](http://jaspar.genereg.net/) ‚Äî Database of transcription factor binding profiles.

### Disease

*   [KEGG DRUG](https://www.genome.jp/kegg/drug/) ‚Äî Comprehensive, approved drug information.
*   [DrugBank](https://go.drugbank.com/) ‚Äî Database of drugs and targets (University of Alberta).

### Interaction

#### Drug-Gene Interaction

*   [DGIdb](https://www.dgidb.org/) ‚Äî Drug-gene interactions and the druggable genome.
*   [Comparative Toxicogenomics Database](http://ctdbase.org/) ‚Äî Chemical-gene interactions, chemical-disease and gene-disease associations, chemical-phenotype associations.
*   [SNAP](https://snap.stanford.edu/biodata/datasets/10002/10002-ChG-Miner.html) ‚Äî Dataset of drug-gene interactions.
*   [Therapeutics Data Commons](https://tdcommons.ai/) ‚Äî Datasets for drug-target, response, drug-drug interaction, etc.

#### Drug (Cell Line) Response

*   [NCI60](https://dtp.cancer.gov/discovery_development/nci-60/) ‚Äî Focuses on 60 cancer cell lines and many drugs.
*   [Genomics of Drug Sensitivity in Cancer (GDSC)](https://www.cancerrxgene.org/) ‚Äî Drug sensitivity for \~1000 human cancer cell lines and hundreds of compounds.
*   [Cancer Cell Line Encyclopedia](https://sites.broadinstitute.org/ccle/) ‚Äî Database of \~1000 cancer cell lines.
*   [CellMiner Cross Database (CellMinerCDB)](https://discover.nci.nih.gov/cellminercdb/) ‚Äî Integrates multiple cancer cell line databases.

#### Chemical-Protein Interaction

*   [STITCH](http://stitch.embl.de/) ‚Äî Chemical-protein interactions.
*   [BindingDB](https://www.bindingdb.org/rwd/bind/index.jsp) ‚Äî Compounds and target database.
*   [PDBBind](https://www.pdbbind-plus.org.cn/) ‚Äî Binding affinity data for biomolecular complexes.
*   [CrossDocked2020](https://arxiv.org/abs/2001.01037) ‚Äî Large-scale dataset for structure-based virtual screening.

#### Protein-Protein Interaction

*   [STRING](https://string-db.org/) ‚Äî PPI networks for multiple organisms.
*   [BioGRID](https://thebiogrid.org/) ‚Äî Protein, genetic, and chemical interactions.
*   [HIPPIE](http://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie/) ‚Äî Human protein-protein interaction database.

#### Knowledge Graph

*   [Drug Mechanism Database (DrugMechDB) (‚≠ê67)](https://github.com/SuLab/DrugMechDB/tree/2.0.1) ‚Äî Mechanisms of action from drug to disease.
*   [DRKG (‚≠ê670)](https://github.com/gnn4dr/DRKG) ‚Äî Large-scale biological knowledge graph for drug discovery.
*   [Hetionet (‚≠ê340)](https://github.com/hetio/hetionet) ‚Äî Heterogeneous network integrating genes, diseases, drugs, pathways, and more.
*   [OpenBioLink (‚≠ê157)](https://github.com/OpenBioLink/OpenBioLink) ‚Äî Benchmark datasets for biological knowledge graph completion.
*   [PrimeKG (‚≠ê697)](https://github.com/mims-harvard/PrimeKG) ‚Äî Multi-modal precision medicine knowledge graph integrating clinical, genetic, and drug data.

### Clinical Trial

*   [ClinicalTrials.gov](https://clinicaltrials.gov/) ‚Äî Privately and publicly funded clinical studies.
*   [ICD10](https://icd.who.int/browse10/2019/en) ‚Äî International Classification of Diseases, 10th revision.
*   [EU Drug Regulating Authorities Clinical Trials DB (EudraCT)](https://eudract.ema.europa.eu/) ‚Äî European clinical trial database.
*   [MIMIC-IV](https://mimic.mit.edu/) ‚Äî Freely accessible critical care database.

***

## API

*   [PubMed E-utilities (esearch/efetch)](https://www.nlm.nih.gov/dataguide/edirect/esearch.html) ‚Äî APIs for searching and retrieving biomedical literature from PubMed.
*   [NCBI E-utilities](https://www.ncbi.nlm.nih.gov/books/NBK25501/) ‚Äî Unified APIs for accessing NCBI databases (Gene, GEO, SRA, PubChem, etc).
*   [UniProt REST API](https://www.uniprot.org/help/api) ‚Äî Programmatic access to protein sequence and functional annotation data.
*   [Ensembl REST API](https://rest.ensembl.org/) ‚Äî API for genomic annotations, variants, genes, and comparative genomics.
*   [KEGG REST API](https://www.kegg.jp/kegg/rest/keggapi.html) ‚Äî API for accessing KEGG pathways, compounds, genes, and reactions.
*   [ChEMBL Web Services](https://www.ebi.ac.uk/chembl/ws) ‚Äî REST API for bioactive molecules, targets, and bioassays.
*   [Open Targets Platform API](https://platform.opentargets.org/api) ‚Äî API for target‚Äìdisease associations integrating genetics, genomics, and drug data.
*   [ClinicalTrials.gov API](https://clinicaltrials.gov/api/gui) ‚Äî API for querying clinical trial metadata and results.

***

## Preprocessing Tools

*   [Chemistry Development Kit (‚≠ê568)](https://github.com/cdk/cdk) ‚Äî Cheminformatics software & machine learning tools.
*   [FlashDeconv (‚≠ê10)](https://github.com/cafferychen777/flashdeconv) ‚Äî High-performance spatial transcriptomics deconvolution (\~1M spots in \~3 min).
*   [RDKit (‚≠ê3.3k)](https://github.com/rdkit/rdkit) ‚Äî Cheminformatics software & machine learning toolkit.
*   [ChatSpatial (‚≠ê11)](https://github.com/cafferychen777/ChatSpatial) ‚Äî MCP server for spatial transcriptomics analysis via natural language.
*   [Scanpy](https://scanpy.readthedocs.io/en/stable/) ‚Äî Python library for scRNA-seq analysis.
*   [Seurat](https://satijalab.org/seurat/) ‚Äî R library for scRNA-seq analysis.
*   [Squidpy](https://squidpy.readthedocs.io/) ‚Äî Python library for spatial single-cell analysis.

***

## Machine Learning Tasks and Models

### Drug Response Prediction

*   [drGAT (‚≠ê1)](https://github.com/inoue0426/drGAT) ‚Äî Attention-based model for drug response prediction with gene explainability.
*   [MOFGCN (‚≠ê6)](https://github.com/weiba/MOFGCN/tree/main) ‚Äî GCN + heterogeneous network.
*   [DeepDSC](https://ieeexplore-ieee-org.ezp2.lib.umn.edu/stamp/stamp.jsp?tp=\&arnumber=8723620\&tag=1) ‚Äî Autoencoder + fully connected NN.
*   [DGDRP (‚≠ê0)](https://github.com/minwoopak/heteronet) ‚Äî Multi-view embedding neural network.
*   [DeepAEG (‚≠ê3)](https://github.com/zhejiangzhuque/DeepAEG) ‚Äî GNN embedding + attention mechanism.

### Drug Repurposing

*   [DeepPurpose (‚≠ê1.1k)](https://github.com/kexinhuang12345/DeepPurpose) ‚Äî Deep learning library for drug repurposing.

### Drug Target Interaction

*   [NeoDTI (‚≠ê77)](https://github.com/FangpingWan/NeoDTI) ‚Äî Library for drug-target interaction prediction.
*   [DTINet (‚≠ê185)](https://github.com/luoyunan/DTINet) ‚Äî Network-based framework integrating heterogeneous biological data for DTI prediction.
*   [DeepDTA (‚≠ê291)](https://github.com/hkmztrk/DeepDTA) ‚Äî Deep learning model using CNNs on protein sequences and drug SMILES.
*   [GraphDTA (‚≠ê292)](https://github.com/thinng/GraphDTA) ‚Äî Graph neural network‚Äìbased DTI prediction using molecular graphs.
*   [MolTrans (‚≠ê224)](https://github.com/kexinhuang12345/MolTrans) ‚Äî Transformer-based DTI model leveraging molecular substructures.
*   [DrugBAN (‚≠ê140)](https://github.com/peizhenbai/DrugBAN) ‚Äî Bilinear attention network for interpretable DTI prediction.

### Compound-Protein Interaction

*   [MCPINN (‚≠ê3)](https://github.com/mhlee0903/multi_channels_PINN) ‚Äî Drug discovery via compound-protein interaction and machine learning.
*   [TransformerCPI (‚≠ê152)](https://github.com/lifanchen-simm/transformerCPI) ‚Äî CPI prediction using Transformer.

### Pre-trained Embedding

*   [Evolutionary Scale Modeling (ESM) (‚≠ê4k)](https://github.com/facebookresearch/esm) ‚Äî Protein embeddings.
*   [ChemBERTa-2 (‚≠ê486)](https://github.com/seyonechithrananda/bert-loves-chemistry) ‚Äî Chemical embeddings & prediction.

### LLM for Biology

*   [AI4Chem/ChemLLM-7B-Chat](https://huggingface.co/AI4Chem/ChemLLM-7B-Chat) ‚Äî LLM for chemical & molecular science.
*   [BioGPT (‚≠ê4.5k)](https://github.com/microsoft/BioGPT) ‚Äî LLM for biomedical text generation.
*   [GeneGPT (‚≠ê420)](https://github.com/ncbi/GeneGPT) ‚Äî LLM for biomedical information, integrated with various APIs.
*   [GenePT (‚≠ê308)](https://github.com/yiqunchen/GenePT) ‚Äî Foundation LLM for single-cell data.
*   [scPRINT (‚≠ê138)](https://github.com/cantinilab/scPRINT) ‚Äî Pretrained on 50M cells for scRNA-seq denoising & zero imputation.

### Foundation Models

*   [scFoundation (‚≠ê390)](https://github.com/biomap-research/scFoundation) ‚Äî Large-scale foundation model for single-cell gene expression, enabling multiple downstream tasks.
*   [scGPT (‚≠ê1.5k)](https://github.com/bowang-lab/scGPT) ‚Äî Transformer-based foundation model pretrained on millions of single-cell profiles.
*   [BulkFormer (‚≠ê41)](https://github.com/KangBoming/BulkFormer) ‚Äî Foundation model for bulk RNA-seq data; learns general transcriptomic representations.

***

